serverless.yml

```diff
@@ -0,0 +1,19 @@
+service: serverless-tunity
```

Comment:
Thus yaml is only for testing locally ?

Reply:
The serverless.yml file defines AWS Lambda structure and deployment. It's used for both local testing and actual deployment.